src/StickyParallaxHeader.js

```diff
@@ -44,6 +44,20 @@ class StickyParallaxHeader extends Component {
     this.props.onRef?.(this)
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    const { headerHeight, parallaxHeight, tabs } = this.props
+    const { currentPage, isFolded } = this.state
+    const isRenderingTabs = tabs && tabs.length > 0
+
+    if (isRenderingTabs && prevState.currentPage !== currentPage && isFolded) {
```

Comment:
Would be great if you could also export this currentPage check to variable 👌

Reply:
exported to external variable